redirect/routes.php

```diff
@@ -597,7 +596,8 @@ class Route {
         'complexity'     => 'comp',
         'sp-ge'          => 'ge-sp',
         'roomfinder'     => 'finder',
-        'patterns'       => 'pat',
+        'patterns'       => 'pse',
+        'pat'            => 'pat',
```

Comment:
This should probably point to pse as well right?

Reply:
Good catch!